lesson04.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    covariance = 1.0 # change this to one.
</pre><p>Great comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>